### PR TITLE
remove structured logger

### PR DESCRIPTION
**Pull request by winsvega**
https://github.com/ethereum/webthree-umbrella/issues/258

### CODE REVIEW

File: eth/main.cpp [hunk 1]
```diff
@@ -387,7 +386,6 @@ int main(int argc, char** argv)
 	bytes extraData;
 
 	/// Structured logging params
-	bool structuredLogging = false;
 	string structuredLoggingFormat = "%Y-%m-%dT%H:%M:%S";
```
Comment (chriseth) [inline eth/main.cpp]: Please also remove these.

**Comment by bobsummerwill** [comment]
What is the rational for removing this?   What does the StructuredLogger do?

**Comment by bobsummerwill** [comment]
Please could you squash the commits and we are good to good.

For my own education, please could you explain what the StructuredLogger was, and why we don't need it anymore?   Thanks!

**Comment by chriseth** [comment]
StructuredLogger was a frontier-mechanism to get similarly structured logs from all client implementation to find consensus and networking issues. These logs were never analized (also because the network was more healthy than expected).

**Pull request merged**